core/internal/gethwrappers/generation/compile_contracts.sh

```diff
@@ -3,7 +3,8 @@
 echo "compiling contracts"
 
 CDIR=$(dirname "$0")
-COMPILE_COMMAND=$(<"$CDIR/compile_command.txt")
+#COMPILE_COMMAND=$(<"$CDIR/compile_command.txt")
+COMPILE_COMMAND=$(yarn && yarn workspace @chainlink/contracts belt compile solc)
```

Comment:
Why?

Reply:
compile_command.txt doesn't exist anymore so the `make go-solidity-wrappers` was failing. Looking at this again I think we can just remove the compile_contracts.sh script also since we compile [here](https://github.com/smartcontractkit/chainlink/blob/6428fd777053e866378cad4be587a6113c2178c3/GNUmakefile#L75) anyways cc @alexcoventry ?

Reply:
on second thought - it looks like compile_command.txt is used in test also, I can add it back but confused why it was removed in https://github.com/smartcontractkit/chainlink/pull/3563/files

Reply:
That was a mistake. I added it back in #3752.